src/features/filter.ts

```diff
@@ -1,10 +1,63 @@
-export const actions = { /* put action creators here */};
+export enum Statuses {
+  All = 'all',
+  Completed = 'completed',
+  Active = 'active',
+}
 
-const filterReducer = () => {
-  return {
+type SetQueryAction = {
+  type: 'filter/SET_QUERY';
```

Comment:
Would be great to create an enum for action types

Reply:
Done